docs/database/usage/index.md

```diff
@@ -335,3 +335,38 @@ and caching. To disable this functionality, update the `database_persistence_ena
 ```
 
 To enable persistence, view the [Offline Support](/database/offline-support) documentation.
+
+# Change the database URL
+
+## Switch the database URL of the default app
+
+If the default installed firebase instance needs to address a different database within the same project, call the database method on the default app with passing the database URL. 
+For example: 
+
+```js
+import firebase from '@react-native-firebase/app';
+import '@react-native-firebase/database';
+
+const database = firebase.app().database('https://path-to-database.firebaseio.com');
```

Comment:
Trying to reduce size/examples, this would be more practical I think:

```
const ref = database('https://path-to-database.firebaseio.com').ref();
```

Reply:
This was the whole point for me and also the reason why I split this section into two different parts:

Changing the database URL for the default Firebase app is not working when calling the `database` module directly.
The `database` module accepts a Firebase instance only, not a simple URL string.
=> `database(FirebaseApp);`

If you want to point your default Firebase app **within the same project** to a different database you need to call the default app first, and then database. Only in this case `database` accepts a simple URL string.
=> `firebase.app().database('http://your-url-string.com')`;

See discussion here: #3049

Maybe I am missing something fundamental on using the lib correctly.
I am happy if you could correct me in this case. Thanks a lot.

Reply:
@lisabaut your original example is correct, can only change it from an app instance